src/components/MoneyTemporaryForRefactorRequestConfirmationList.tsx

```diff
@@ -641,11 +634,29 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
                 {button}
             </>
         );
-    }, [isReadOnly, iouType, selectedParticipants.length, confirm, bankAccountRoute, iouCurrencyCode, policyID, splitOrRequestOptions, formError, styles.ph1, styles.mb2]);
+    }, [isReadOnly, iouType, selectedParticipants.length, confirm, bankAccountRoute, iouCurrencyCode, policyID, splitOrRequestOptions, formError, styles.ph1, styles.mb2, isTypeSplit]);
 
     // An intermediate structure that helps us classify the fields as "primary" and "supplementary".
     // The primary fields are always shown to the user, while an extra action is needed to reveal the supplementary ones.
     const classifiedFields = [
+        {
+            item: (
+                <MenuItem
+                    label={translate('moneyRequestConfirmationList.paidBy')}
+                    interactive={!transaction?.isFromGlobalCreate}
```

Comment:
Yes because when we create split request from global, maybe we cannot know the current user accountID of other user. So I'm disabling this.

@youssef-lr What do you think about this?

Reply:
>maybe we cannot know the current user accountID of other user

Good point, but this is only true if we're splitting with new accounts as we'll only have their optimistic accountIDs. We can handle this in the backend though so it should be fine to just send the optimistic accountID, we'll use the real accountID once it's created in the backend.

So to answer your question @nkdengineer we should still allow selection of payer from Global Create.